### PR TITLE
chore: remove unused files

### PR DESCRIPTION
**Pull request by uladzimirdev**
### Description

drop files 

Result of running knip on our repo

### CODE REVIEW

**Comment by nemanjaglumac** [comment]
image

**Review by nemanjaglumac — APPROVED** [review]

File: frontend/src/metabase/visualizations/components/settings/ChartSettingMessage.tsx [hunk 1]
```diff
@@ -1,20 +0,0 @@
-import type { PropsWithChildren } from "react";
-
-import { Center, useMantineTheme } from "metabase/ui";
-
-export const ChartSettingMessage = ({ children }: PropsWithChildren) => {
```
Comment (uladzimirdev) [inline frontend/src/metabase/visualizations/components/settings/ChartSettingMessage.tsx:5]: @oisincoveney can we safely remove it?

File: frontend/src/metabase/components/Popover/constants.ts [hunk 2]
```diff
@@ -1 +0,0 @@
-export const DEFAULT_Z_INDEX = 400;
```
Comment (uladzimirdev) [inline frontend/src/metabase/components/Popover/constants.ts:1]: @rafpaf do we need to keep it?

File: frontend/src/metabase/visualizations/components/settings/ChartSettingMessage.tsx (hunk 1 above)
Comment (oisincoveney) [reply frontend/src/metabase/visualizations/components/settings/ChartSettingMessage.tsx:5]: I don't think so - we use it in:

```
frontend/src/metabase/visualizations/components/settings/ChartSettingSeriesOrder.tsx:18:10:..
frontend/src/metabase/visualizations/components/settings/ChartSettingSeriesOrder.tsx:272:10..
frontend/src/metabase/visualizations/components/settings/ChartSettingSeriesOrder.tsx:272:53..
frontend/src/metabase/visualizations/components/settings/ChartSettingOrderedSimple.tsx:12:1.. frontend/src/metabase/visualizations/components/settings/ChartSettingOrderedSimple.tsx:118:..
frontend/src/metabase/visualizations/components/settings/ChartSettingOrderedSimple.tsx:118:.
```

At least according to the `get references` command on neovim

Comment (uladzimirdev) [reply frontend/src/metabase/visualizations/components/settings/ChartSettingMessage.tsx:5]: isn't it come from 

```
import {
  ChartSettingMessage,
  ChartSettingOrderedSimpleRoot,
} from "./ChartSettingOrderedSimple.styled";
```

But the name is the same. I'm asking you as you've recently touched it

**Review by iethree — APPROVED** [review]
🤩

File: frontend/src/metabase/visualizations/components/settings/ChartSettingMessage.tsx (hunk 1 above)
Comment (oisincoveney) [reply frontend/src/metabase/visualizations/components/settings/ChartSettingMessage.tsx:5]: Ah, I'm on the branch that's actually using that component ([this one](https://github.com/metabase/metabase/pull/51332/files)) . That will delete `ChartSettingOrderedSimple.styled` and use this component

Comment (oisincoveney) [reply frontend/src/metabase/visualizations/components/settings/ChartSettingMessage.tsx:5]: Let's merge this PR, and then I'll put it back in my PR where it belongs

File: frontend/src/metabase/components/Popover/constants.ts (hunk 2 above)
Comment (rafpaf) [reply frontend/src/metabase/components/Popover/constants.ts:1]: The only Z_INDEX value I care about is OVERLAY_Z_INDEX

It looks like I messed up, though. We have

```
frontend/src/metabase/css/core/overlays/constants.tsx
1:export const OVERLAY_Z_INDEX = 200;

frontend/src/metabase/css/core/overlays/constants.ts
1:export const OVERLAY_Z_INDEX = 200;
```

Could you please remove `frontend/src/metabase/css/core/overlays/constants.tsx` as part of this work?

**Review by oisincoveney — APPROVED** [review]
😍

File: frontend/src/metabase/components/Popover/constants.ts (hunk 2 above)
Comment (rafpaf) [reply frontend/src/metabase/components/Popover/constants.ts:1]: No problem if you don't want to (because of CI or some other reason)

Comment (uladzimirdev) [reply frontend/src/metabase/components/Popover/constants.ts:1]: I wanted to, but I was afraid to do it, thanks for confirming!

File: frontend/src/metabase/visualizations/components/settings/ChartSettingMessage.tsx (hunk 1 above)
Comment (uladzimirdev) [reply frontend/src/metabase/visualizations/components/settings/ChartSettingMessage.tsx:5]: I can revert this file, no problem

**Pull request merged**